src/web/core/src/main/java/org/geoserver/web/data/resource/DimensionEditorBase.java

```diff
@@ -417,9 +424,82 @@ protected void onUpdate(AjaxRequestTarget target) {
+            checkResult = true;
+        }
+        final WebMarkupContainer fixedValueRangeContainer =
+                new WebMarkupContainer("fixedValueRangeContainer");
+        fixedValueRangeContainer.setOutputMarkupId(true);
+
+        fixedValues = new TextArea<>("fixedValues", new PropertyModel<>(model, "fixedValues"));
+        fixedValues.setVisible(checkResult);
+        fixedValueRangeContainer.add(fixedValues);
+        fixedValues.add(
+                (IValidator<String>)
+                        validatable -> {
+                            try {
+                                checkFixedValueRange(validatable.getValue(), getAttributeType());
+                            } catch (Exception e) {
+                                String messageKey = "invalidFixedValueRange";
+                                validatable.error(
+                                        new ValidationError(messageKey)
+                                                .addKey(messageKey)
+                                                .setVariable("actual", validatable.getValue()));
+                            }
+                        });
+        fixedValueCheckbox = new CheckBox("fixedValue", Model.of(checkResult));
+        configs.add(fixedValueCheckbox);
+        fixedValueCheckbox.add(
+                new AjaxFormComponentUpdatingBehavior("click") {
+
+                    @Override
+                    protected void onUpdate(AjaxRequestTarget target) {
+                        Boolean visible = fixedValueCheckbox.getModelObject();
+                        fixedValues.setVisible(visible);
+                        target.add(fixedValueRangeContainer);
+                    }
+                });
+        configs.add(fixedValueRangeContainer);
+
         initComponents();
     }
 
+    public boolean isDouble(String value) {
+        try {
+            Double.parseDouble(value);
+            return true;
+        } catch (NumberFormatException e) {
+            return false;
+        }
+    }
+
+    protected void checkFixedValueRange(String fixedValues, Class<?> dataType) {
+        List<String> fixedValueList = Arrays.asList(fixedValues.split(","));
+        if (!fixedValueList.isEmpty()) {
+            for (String fixedValue : fixedValueList) {
+                if (Number.class.isAssignableFrom(dataType)) {
+                    boolean checkresult = isDouble(fixedValue);
+                    if (!checkresult) {
+                        throw new IllegalArgumentException(
+                                "Invalid Fixed Value range specification");
+                    }
+                } else if (Date.class.isAssignableFrom(dataType)) {
+                    TimeParser parser = new TimeParser();
+                    try {
+                        parser.parse(fixedValue);
+                    } catch (ParseException e) {
+                        LOGGER.log(Level.WARNING, e.getMessage(), e);
```

Comment:
If we have a Parse exception probably the validation did not pass. This should not be simply catched.